packages/contracts/test/registry/dao-registry.ts

```diff
@@ -140,4 +141,86 @@ describe('DAORegistry', function () {
       .to.be.revertedWithCustomError(ensSubdomainRegistrar, 'AlreadyRegistered')
       .withArgs(daoDomainHash, ensSubdomainRegistrar.address);
   });
+
+  // without mocking we have to repeat the tests here to make sure the validation is correct
+  describe('name validation', () => {
+    it('should validate the passed name correctly (< 32 bytes long name)', async () => {
+      const baseName = 'this-is-my-super-valid-name';
+
+      // loop through the ascii table
+      for (let i = 0; i < 127; i++) {
```

Comment:
Isn't it possible to still pass char values ranging from 128 to 255? The ENS check should fail anyway